src/core/nexus/HY_PointHydroNexusRemote.cpp

```diff
@@ -102,7 +103,7 @@ double HY_PointHydroNexusRemote::get_downstream_flow(std::string catchment_id, t
             /* request      = */ &stored_sends.back().mpi_request);
 
         //std::cerr << "Remote send from rank " << world_rank << " to rank " << iter->second << " on tag " << tag << std::endl;
-
+        return (double)SENTINEL_IS_REMOTELY_HANDLED;
```

Comment:
Previously, this function wasn't explicitly returning anything.  That was valid, and somewhat understandable (this was the condition when the output should be sent to a remote instead of written locally), but it resulted in a warning that was contributing to lots of compiler output noise.  It also happened to be very close in proximity to how the output is handled, so it made sense to use a specific value both for improving general internal operation and to help debug the issue of duplicate writes that we are seeing.